.github/pull_request_template.md

```diff
@@ -1,14 +1,14 @@
-## Description
-
 <!-- Insert PR description-->
 
-## Does this PR need a docs update or release note?
+---
+
+#### Does this PR need a docs update or release note?
 
 - [ ] :white_check_mark: Yes, it's included
 - [ ] :clock1: Yes, but in a later PR
-- [ ] :no_entry: No 
+- [ ] :no_entry: No
 
-## Type of change
+#### Type of change
```

Comment:
There is no `####` in markdown.  3 hashes is the maximum.  Also, (not a strongly held opinion) I'd recommend keeping with two.

Reply:
[GFM](https://github.github.com/gfm/#atx-headings) - the flavor of markdown GitHub uses, does have up to 6. The reason why I moved away from 2 to 4 was primarily because of the heading underlines, but we can drop back to 2. The main reason for bringing this up was the `## Description` block.

|Before|After|
|---|---|
|![2023-02-10-14-44-06](https://user-images.githubusercontent.com/14259816/218052163-808f3ea8-af66-47d4-918c-2a157433a658.png)|![2023-02-10-14-47-48](https://user-images.githubusercontent.com/14259816/218052501-efb7cbcb-69da-4eb2-8b42-04c68bb37ff1.png)|